classes/01class/exercises/c01-aws01/mail2kvanitha/README.md

```diff
@@ -0,0 +1,42 @@
+## EC2 (c01-aws01)
+
+- Use the CLI to create a new EC2 Instance inside the default VPC with a public IP address and connect to it though SSH.
+- Create another EC2, also using the CLI instance in the same VPC but with private address only and connect to it.
+Steps to connect from one instance with Public IP to another instance with no Public IP.
+STEP1:CREATE SECURITY GROUP:
+----------------------------
+aws ec2 create-security-group --group-name webserver-sg --description "Webserver Security Group"
```

Comment:
can you provide what rules have been created in this security group and which command was used to create those rules?

Reply:
Updated the README.md with required security groups and rules added.